core/src/main/java/io/grpc/inprocess/InProcessServerBuilder.java

```diff
@@ -82,4 +84,9 @@ private InProcessServerBuilder(String name) {
   protected InProcessServer buildTransportServer() {
     return new InProcessServer(name);
   }
+
+  @Override
+  public InProcessServerBuilder useTransportSecurity(File certChain, File privateKey) {
+    throw new UnsupportedOperationException("TLS not supported in InProcessServer");
```

Comment:
FYI: It _may_ be reasonable to ignore the call. I think this is fine for now, though.